internal/cli/cmd/install.go

```diff
@@ -89,6 +90,13 @@ func newCmdUninstall() *cobra.Command {
 		Short: "Uninstall Cilium",
 		Long:  ``,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			h := hubble.NewK8sHubble(k8sClient, hubble.Parameters{
+				Namespace: params.Namespace,
+				Writer:    params.Writer,
+			})
+			if err := h.Disable(context.Background()); err != nil {
+				fatalf("Unable to disable Hubble:  %s", err)
```

Comment:
Not sure whether we should fatal here, e.g. in case Hubble wasn't enabled previously. We should still make sure to uninstall Cilium in case we fail to disable Hubble.

Reply:
Make sense, although arguably `hubble.Disable()` should not return an error when hubble is not enabled. I expect an error to be returned when something fatal happen (e.g. cannot connect to the cluster) that would prevent cilium to be uninstalled as well.

Note that, in the current implementation, there is no path were `h.Disable()` can return a non-nil error.

Reply:
Fair enough. I was slightly confused by the second commit in this PR which makes `(*K8sHubble).Disable` return the error from `(*K8sHubble).disableRelay` but I didn't realize that the latter currently also always returns `nil`. Let's leave it as is then.